Add the loss variable distributions

orelgueta opened the pull request.

TarekHC commented:
Hi @orelgueta 

Does this mean we should use from now on "MLP_tanh" instead "MLP_small"?

For the moment is not that important really... But want to be up to date! :D

Tarek

orelgueta commented:
Actually, we switched now to MLP_logistic, if you look at the latest plots I sent you (and my presentation at the ASWG meeting a few weeks ago), you will see that it now performs better than the tanh one.
I already modified export_event_types.py accordingly.
